Home work #1 is done

alinaternova opened the pull request:
- calculator named as CalcWithSeparateMethods.java in main > Hometasks folder;
  - tests for it in tests > Hometasks folder.

alinaternova committed:
Home work #1 is done:
  - calculator named as CalcWithSeparateMethods.java in main > Hometasks folder;
  - tests for it in tests > Hometasks folder.

AutomationSquad commented on src/main/java/Hometasks/CalcWithSeparateMethods.java:
```diff
@@ -0,0 +1,127 @@
+    public static int intAddition (Integer arg1, Integer arg2) {
+        int sum = 0;
+        sum = arg1 + arg2;
+        return sum;
```
можна замінити це 
` int sum = 0; sum = arg1 + arg2; return sum;` 
на 
`return arg1 + arg2`
для наступних методів по аналогії

alinaternova replied:
Зрозуміла, дякую за фідбек!

alinaternova replied:
Виправлю.

AutomationSquad reviewed (commented):
Все ок) тільки на майбутнє заливай тільки домашку, без лівих файлів)

alinaternova committed:
Home work #1 is done:
  - calculator named as CalcWithSeparateMethods.java in main > Hometasks folder;
  - tests for it in tests > Hometasks folder.
